### PR TITLE
LPD-30098 Technical Task | Optimize PublicationsUserCollaboration#InviteUserAsAdmin

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for: [LPD-30098](https://liferay.atlassian.net/browse/LPD-30098)

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:test:publications

File: modules/test/playwright/tests/change-tracking-web/publicationsUserCollaboration.spec.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+	await page
+		.getByLabel('Delete- Loading')
+		.getByRole('button', {name: 'Delete'})
+		.click();
+
+	await waitForSuccessAlert(
+		page,
+		'Success:Your request completed successfully.'
+	);
```
Comment (pei-jung) [inline modules/test/playwright/tests/change-tracking-web/publicationsUserCollaboration.spec.ts:165]: Add `com.liferay.journal.service.impl.JournalArticleServiceImpl#deleteArticle` to `JSONWebServicesJournalApiHelper` to delete the web content.

**Review by pei-jung — CHANGES_REQUESTED** [review]

**Comment by gislayne-vitorino** [comment]
ci:test:sf

File: modules/test/playwright/tests/change-tracking-web/publicationsUserCollaboration.spec.ts (hunk 1 above)
Comment (gislayne-vitorino) [reply modules/test/playwright/tests/change-tracking-web/publicationsUserCollaboration.spec.ts]: Now the web content is being added within a site, that gets deleted afterwards, so there's no need to add this method anymore

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
Failures might be related to the frontend changes mentioned [here](https://liferay.slack.com/archives/C1SJ64S9X/p1721932336510469)

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by pei-jung** [comment]
@gislayne-vitorino good work! See my changes in https://github.com/liferay-publications/liferay-portal/pull/1319.

**Pull request closed without merging**